constraint/pkg/client/drivers/rego/driver_unit_test.go

```diff
@@ -743,6 +744,7 @@ func TestDriver_ExternalData(t *testing.T) {
 
 			d, err := New(
 				AddExternalDataProviderCache(externaldata.NewCache()),
+				AddExternalDataProviderResponseCache(externaldata.NewProviderResponseCache(context.Background(), 1*time.Minute)),
```

Comment:
would we want this to be configurable?

Reply:
TTL? It is configurable - https://github.com/open-policy-agent/gatekeeper/pull/2823/files#diff-2873f79a86c0d8b3335cd7731b0ecf7dd4301eb19a82ef7a1cba7589b5252261R112

Reply:
I missed that this is for tests